core/src/sql/statements/define/mod.rs

```diff
@@ -47,15 +45,14 @@ pub enum DefineStatement {
 	Database(DefineDatabaseStatement),
 	Function(DefineFunctionStatement),
 	Analyzer(DefineAnalyzerStatement),
-	Token(DefineTokenStatement),
-	Scope(DefineScopeStatement),
```

Comment:
Not sure if this is safe - CF stores statements for tables and databases, so this may be affected for anyone using CF? Since both are tied with database.

Reply:
I don't understand exactly what you mean. Can you provide an example of how this change may affect CF users?